src/main/java/org/llorllale/cactoos/matchers/MatcherOf.java

```diff
@@ -46,9 +46,7 @@
  *
  * @param <T> Type of object to match
  * @since 0.12
- * @todo #135:30min Remove all constructors except the last one so that
- *  every matcher implemented using MatcherOf take care of properly
- *  describe itself and the mismatch.
+ * @todo #165:30m Rewrite the Matcher's, which constructed from others constructor, to use the last constructor of MatcherOf. And delete all constructors except last one.
```

Comment:
@ruca905 I think you need to add some line return, qulice won't be happy if the line is too long